README.md

```diff
@@ -2,16 +2,15 @@ imageproc
 ====
 
 ![crates.io](https://img.shields.io/crates/v/imageproc.svg)
-[![Join the chat at https://gitter.im/PistonDevelopers/imageproc](https://badges.gitter.im/PistonDevelopers/imageproc.svg)](https://gitter.im/PistonDevelopers/imageproc?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
I'm not quite sure how permissions for creating chat rooms are handled with `gitter` but if you want one at `gitter.im/image-rs` then we can surely find out.

Reply:
The imageproc room hasn't seen a lot of activity and I no longer monitor it, so for now I'm happy to just drop gitter support.